circleci/dockercfg.enc

```diff
@@ -0,0 +1 @@
+2�Y�w��G��c5�m�d7�j�����i�7>�;+�tc�@�����W|�+��&E�����Ο<`D9Fϑ#����ܔ	즦㒌��Odx�Jt�E
ҧ��.'�U���G���bǩl��u�B�񧨛�ܨ|�
```

Comment:
It should be binary file ...

Reply:
What do you want me to do?

Reply:
`*.enc` is already set as binary: https://github.com/mapfish/mapfish-print/blob/master/.gitattributes#L15

Reply:
*.enc binary
In .gitattribute but it's already done, then I don't know, keep it like it.

Reply:
But github doesn't care about gitattributes: https://stackoverflow.com/a/24382933/829301

Reply:
ok :-/